src/foam/nanos/notification/email/ApplyBaseArgumentsEmailPropertyService.js

```diff
@@ -92,6 +92,14 @@
         templateArgs.put("privacyUrl", url + appConfig.getPrivacyUrl());
         templateArgs.put("privacyLabel", appConfig.getPrivacy());
 
+        // Temporary color until token support is added for email
+        templateArgs.put("primary1", theme.getPrimary1());
```

Comment:
you can proceed these with theme.
theme.primary1
and use {{theme.primary1}} later.  
This would be preferable, as if find we need more theme support, then we can just get all axioms and all them all to the args, and the 'theme.' prefix will avoid name collisions.

Reply:
done